### PR TITLE
Fix href pointing to RespaAdmin in Navbar (TAM-144)

### PR DESCRIPTION
**Pull request by urshala**
The link to RespaAdmin was always pointing to production despite
running in development mode. Fix it so that the  link points to
QA respaAdmin if it is running in development mode.

### CODE REVIEW

**Review by wiltso — CHANGES_REQUESTED** [review]
Looks good change wise, could you just make sure the code style is fixed and the tests would pass

File: src/domain/header/MainNavbarContainer.js [hunk 1]
```diff
@@ -9,6 +9,11 @@ import MainNavbar from './MainNavbar';
 export const selector = createStructuredSelector({
   isAdmin: isAdminSelector,
   isLoggedIn: isLoggedInSelector,
+  respaAdminUrl: () => {
```
Comment (wiltso) [inline src/domain/header/MainNavbarContainer.js:12]: With this assignment `respaAdminUrl` will be a function that returns a string and not just a string.

**Review by wiltso — COMMENTED** [review]
Could you also fix the code style in the unchanged files that the github action is warning about.

File: src/domain/header/MainNavbarContainer.js (hunk 1 above)
Comment (urshala) [reply src/domain/header/MainNavbarContainer.js:12]: Ah actually its the return value from the function that will be the type while using selector.

Comment (wiltso) [reply src/domain/header/MainNavbarContainer.js:12]: Yea but when is the function called? As [here](https://github.com/andersinno/varaamo/pull/53/files#diff-f7b6691ae5a9a4525951600e3af070e73c6cb040ac09734c7ca0be2c6a2fffd9R83) you are just passing the function to the `NavItem` component.

Comment (urshala) [reply src/domain/header/MainNavbarContainer.js:12]: This function will be called automatically by `reselect` https://github.com/reduxjs/reselect

**Review by wiltso — APPROVED** [review]
Looks good :+1:

**Pull request merged**